src/dotnet-core-uninstall/LocalizableStrings.resx

```diff
@@ -282,4 +267,31 @@
   <data name="ListX86OptionDescription" xml:space="preserve">
     <value>List x86 .NET Core SDKs or Runtimes.</value>
   </data>
+  <data name="DryRunOptionDescription" xml:space="preserve">
+    <value>Display .NET Core SDKs and Runtimes that will be removed.</value>
+  </data>
+  <data name="YesOptionDescription" xml:space="preserve">
+    <value>Execute the command without requiring Y/n confirmation.</value>
+  </data>
+  <data name="DryRunOutputFormat" xml:space="preserve">
+    <value>*** DRY RUN OUTPUT
+Specified versions:
+{0}
+*** END DRY RUN OUTPUT
+
+To avoid breaking Visual Studio or other problems, read aka.ms/dotnet-core-uninstall.
```

Comment:
Sounds a bit too casual to me

Reply:
This has been run through @KathleenDollard.